compatibility-verifier/compCheck.sh

```diff
@@ -112,14 +121,18 @@ function stopService() {
 # Starts a Pinot cluster given a specific target directory
 function startServices() {
   dirName=$1
+  local CONTROLLER_CONF=$2/ControllerConfig.conf
```

Comment:
Not local, move this up in the script to make it non-local (remove the local keyword). You can put it in line 42

Reply:
updated